3D-microservices-observability/node-js-app/loadgen.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Send requests every $1 seconds"
+
+while [ true ]
+do
+	quantity=$((1 + RANDOM % 20))
+	curl_cmd="curl -X POST http://localhost:3000/shop/order -H 'Content-Type: application/json' -d '{\"styleName\": \"beachops\", \"quantity\": '$quantity'}'"
+	echo $curl_cmd
+	eval $curl_cmd
+	sleep $1
+	printf "\n"
+done
```

Comment:
What happens if developer runs this file without an argument?
Should be nice to check for it and echo some "usage" text.

Reply:
it won't sleep if the developer runs this file without an argument

Reply:
That's a problem.